Makefile

```diff
@@ -497,7 +517,7 @@ WORLD_SUBDIRS+=cddl/usr.sbin/${dir}
 # Project local tools
 .for dir in ${WORLD_SUBDIRS_ZROUTER}
 # Prepend reverse path, then buildworld can go out of source tree
-WORLD_SUBDIRS+=${SRCROOTUP}/${ZROUTER_ROOT}/${dir}
+#WORLD_SUBDIRS+=${SRCROOTUP}/${ZROUTER_ROOT}/${dir}
```

Comment:
IIRC it was added to be able to define extra subdirs that have to be built with FreeBSD buildworld.
Why do you comment it out?